### ISSUE TITLE
Add a mask attribute to <label>

### ISSUE THREAD

**Issue by virginiaMae** [Doenet/DoenetML#362]
create a small, opaque white rectangle behind the label region so that labels always stay legible. 
This attribute should be "true" by default, with the option to set `mask="false"`

**Comment by siefkenj** [comment]
I agree with making the labels have a background. Why do we need an attribute to turn the background off?

**Comment by dqnykamp** [comment]
> I agree with making the labels have a background. Why do we need an attribute to turn the background off?

I don't know, as I haven't thought about that yet. I guess it depends if someone can come up with a use case where having the opaque background is going to cause problems.

We don't have the notion of a publisher file in Doenet where one can specify presentation options.
